packages/lib/src/core/core.ts

```diff
@@ -121,22 +123,40 @@ class Core implements ICore {
         return Promise.resolve(this);
     }
 
+    private async fetchLocaleTranslations(): Promise<Translations> {
+        try {
+            const translation = await getTranslations(
```

Comment:
Small point: when passing so many arguments - it's better to clean up the function signature a bit by passing a single object and extract the props in `get-translations.ts`